src/Site_Command.php

```diff
@@ -607,6 +610,24 @@ public function list_( $args, $assoc_args ) {
 			$where['site_id'] = $assoc_args['network'];
 		}
 
+		if ( isset( $assoc_args['site__user_in'] ) ) {
+			$user = get_user_by( 'login', $assoc_args['site__user_in'] );
```

Comment:
`_in` implies the argument supports a CSV of users. Should we accommodate that here?

Also, we should use `WP_CLI\Fetchers\User` instead of directly calling `get_user_by()`. It will handle the "invalid user" case for us.

Reply:
Il'll take a look at `WP_CLI\Fetchers\User`.

Reply:
I've updated the code to use `WP_CLI\Fetchers\User` and also updated the name of the flag to `--user`.

Reply:
@danielbachhuber I needed to change the flag to `site_user` since it was not properly working with `user`. I guess it conflicts with some global flag or something.

Reply:
> I needed to change the flag to `site_user` since it was not properly working with `user`. I guess it conflicts with some global flag or something.

@marksabbath Yep, it does. `site_user` is a fine alternative.